Content.Shared/Movement/Components/JumpComponent.cs

```diff
@@ -0,0 +1,40 @@
+using Content.Shared.Actions;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+
+namespace Content.Shared.Movement.Components;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class JumpComponent : Component
+{
+    /// <summary>
+    /// How powerful the jump will be when activated
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public float JumpPower = 5f;
+
+
+    // Logical variables
+    [DataField, AutoNetworkedField]
+    public EntProtoId Action = "ActionGravityJump";
+
+    [DataField, AutoNetworkedField]
+    public EntityUid? ActionEntity;
+
+    [DataField, AutoNetworkedField]
+    public bool IsClothing;
+
+    [DataField, AutoNetworkedField]
+    public EntityUid? OnClothingEntity;
+
+
```

Comment:
> Just use itemactiongrant or if not suitable make a clothing variant instead of more action boilerplate code.

but I cant use itemactiongrant because there are accesses there and no way to get variables from it, and I didnt understand the second one about clothing =(

Reply:
> Just use itemactiongrant or if not suitable make a clothing variant instead of more action boilerplate code.

nvm I think I fixed it